### PR TITLE
Da/tab panel

### PR DESCRIPTION
**Pull request by danamansana**
This branch makes two accessibility-related changes: 1) results now are aria-labelled by the tab being shown, and 2) keydown on a selected tab focuses on the results list, rather than an individual result

### CODE REVIEW

**Comment by ktp242** [comment]
Looking good. Just some questions and suggestions.

1. tabbing out the facets will also first focus on the whole result wrapper. Is it expected?
2. the indent of <ol> it should be aligned with <ReturnLink>, and {results} should be just two spaces behind <ol>
3. I feel it should actually be a <ul> because which result is first doesn't affect the application, but it is debatable and minor.
4. bigger issue I think here is actually naming. I suggest not using `results` as the name of the HTML <ol>. Because there's already a `results` on line 243.  I think we could name it `resultsOlElement` or something to indicate it is an HTML element. So is the props naming.
5. I think we should have a prop type check for `propTypes.results` in TabItem.jsx

**Comment by LucasCioffi** [comment]
@ktp242 great comments.  The only clarification I'd add is that @wlla would prefer the search results as an ordered list based on her guidelines in the description of a ticket that was closed before you joined the project (so you wouldn't have known about it) https://jira.nypl.org/browse/WWW-493
"Search results should be organized in an ordered list."

**Comment by danamansana** [comment]
I've addressed the issues raised by Kang above. As to 1, yes we do want to focus the whole wrapper, as I understand it. This is the SCC pattern.

File: src/app/components/TabItem/TabItem.jsx [hunk 1]
```diff
@@ -84,8 +84,8 @@ class TabItem extends React.Component {
          this.focusTab(targetTabIndex);
        } else if (targetTabIndex === "down") {
          let nextElement;
-         if (document.getElementsByClassName("gs-resultsItem-link")[0]) {
-           nextElement = document.getElementsByClassName("gs-resultsItem-link")[0];
+         if (this.props.results()) {
```
Comment (ktp242) [inline src/app/components/TabItem/TabItem.jsx]: Scores for using the React way!

File: src/app/components/TabItem/TabItem.jsx [hunk 2]
```diff
@@ -176,6 +176,7 @@ TabItem.propTypes = {
   selectedFacet: PropTypes.string,
   saveSelectedTabValue: PropTypes.func,
   searchBySelectedFacetFunction: PropTypes.func,
+  results: PropTypes.func,
```
Comment (ktp242) [inline src/app/components/TabItem/TabItem.jsx]: But I think the type is an array not a function.

**Pull request merged**